### PR TITLE
Update angsd

### PR DESCRIPTION
**Pull request by isinaltinkaya**
Update angsd tarball link to official stable release tarball at popgen.dk website.

### CODE REVIEW

**Comment by isinaltinkaya** [comment]
@BiocondaBot please add label

File: recipes/angsd/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,13 @@
-{% set version = "0.939" %}
-{% set sha256 = "43bbc02d964eea59508230a63f44b43d58be5d7ef450064ce5eebd0b0ba3b9dc" %}
```
Comment (bgruening) [inline recipes/angsd/meta.yaml:2]: You downgraded the version?

Comment (isinaltinkaya) [reply recipes/angsd/meta.yaml:2]: Hi,

I am in the ANGSD development team and I am a PhD student working with Thorfinn Korneliussen (who is the main developer of ANGSD). The stable release tarballs of ANGSD are the version numbers that are even, and mainly distributed from popgen.dk website. Therefore we would like to update (downgrade) the angsd bioconda recipe to the latest stable release (0.938) and the link to the main website (popgen.dk). Thanks!

Best regards,
Isin

Comment (bgruening) [reply recipes/angsd/meta.yaml:2]: @isinaltinkaya that makes sense. But people installing angsd without a version will still get the latest, e.g. the beta version. So maybe you can bump the version on github once and then increase the number here?

Comment (isinaltinkaya) [reply recipes/angsd/meta.yaml:2]: I am not sure if I understand- can you explain what you mean by "bump the version on github once and then increase the number here?"? Thanks!

Comment (isinaltinkaya) [reply recipes/angsd/meta.yaml:2]: @bgruening we have just released 0.940 which I included in this PR. Is there anything else I should do?

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**